### PR TITLE
Feature/preapre new release

### PR DESCRIPTION
**Pull request by PerrineAguiar**
PR preparing the new release of the app.

### CODE REVIEW

**Comment by adamantivm** [comment]
@PerrineAguiar one thing, and sorry I missed this in previous PR's, note that if you change the name of the tango_ros_node module, you will change the name with which the library package and artifact is released, leaving behind the older version with no further updates in the repo. Is that the intended behavior? Did you consider using the same package and artifact name with the new implementation?

File: TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/AboutActivity.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ protected void onCreate(Bundle savedInstanceState) {
         TextView textViewVersion = (TextView) findViewById(R.id.text_version);
         try {
             PackageInfo packageInfo = getPackageManager().getPackageInfo(getPackageName(), 0);
-            textViewVersion.setText(packageInfo.versionName + "-" + packageInfo.versionCode);
+            textViewVersion.setText(packageInfo.versionName);
```
Comment (jubeira) [inline TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/AboutActivity.java]: This doesn't show the patch version, is that OK?

Comment (PerrineAguiar) [inline TangoRosStreamer/app/src/main/java/eu/intermodalics/tango_ros_streamer/activities/AboutActivity.java]: It was actually an error, we were using the version code as the patch version, which is wrong. The patch version should be included in the version name and is totally independent from the version code.

**Comment by PerrineAguiar** [comment]
@adamantivm @jubeira Thanks for your comments.
I agree that changing the library name is not ideal but as @jubeira said, the new name reflects better what it is. To have the new functionalities, updating the module name and the class is necessary. Do you think it is a big issue? I am asking you because you are the only one I know using the tango_ros_node.
I will update the wiki entry as soon as this is merged.

**Comment by adamantivm** [comment]
I think it's not a big issue, especially since we're the only ones using it
(that we know) so far, so it's a good time to change before it has more
uses.

**Comment by PerrineAguiar** [comment]
@smits ptal

**Pull request merged**